test/output_tests/output.graphql

```diff
@@ -1 +1 @@
-{"{\n  author {\n    firstName\n    lastName\n  }\n}\n":9,"query otherQuery {\n  person {\n    firstName\n    lastName\n  }\n}\n":10}
\ No newline at end of file
+{"query firstQuery {\n  author {\n    firstName\n    lastName\n  }\n}":9,"query secondQuery {\n  person {\n    firstName\n    lastName\n  }\n}":10}
```

Comment:
persistgraphql doesn't handle queries without names properly.  This isn't an issue for our code because all of our GraphQL operations are required to use unique names.  I've updated the fixtures to work around the issue.